src/osemgrep/core/Metrics_.mli

```diff
@@ -1,9 +1,10 @@
 (*
    Configures metrics upload.
 
-   On - Metrics always sent
-   Off - Metrics never sent
-   Auto - Metrics only sent if config is pulled from the server
+   On   - Metrics always sent
+   Off  - Metrics never sent
+   Auto - Metrics only sent if config is pulled from the registry
```

Comment:
thought – should usage of `--pro` constitute using the app?

Reply:
probably. I don't know exactly the rule either ... there's still some TODOs regarding this is_using_app